app/views/posts/show.html.erb

```diff
@@ -19,6 +19,10 @@
                 </p>
             </div>            
         <% end %>
-    </div>   
+    </div>
+    <div class="button-user">
+        <%= button_to "Add Comment", new_user_post_comment_path(@post.author, @post), method: :get, class: "button" %>
+        <%= button_to "Add Like", new_user_post_like_path(@post.author, @post), method: :get, class: "button" %>
```

Comment:
- **[OPTIONAL]** Instead of redirecting the user to a new form just to like a post, I think we can simply implement the like here, by submitting the action to the create route directly.